src/test/java/io/nats/client/impl/TLSConnectTests.java

```diff
@@ -96,7 +96,11 @@ public void testSimpleTLSConnection() throws Exception {
     @Test
     public void testSimpleTlsFirstConnection() throws Exception {
         if (TestBase.atLeast2_10_3(ensureRunServerInfo())) {
-            try (NatsTestServer ts = new NatsTestServer("src/test/resources/tls_first.conf", false)) {
+            try (NatsTestServer ts = new NatsTestServer(
+                NatsTestServer.builder()
+                    .configFilePath("src/test/resources/tls_first.conf")
+                    .connectValidateTlsFirstMode())
```

Comment:
The newest server runner validates connection in a different way, so it needs to know when it's tls first.